crates/tauri-plugin-holochain-service-consumer/build.rs

```diff
@@ -1,11 +1,4 @@
-const COMMANDS: &[&str] = &[
-    "connect",
-    "install_app",
-    "enable_app",
-    "is_app_installed",
-    "ensure_app_websocket",
-    "sign_zome_call",
-];
+const COMMANDS: &[&str] = &["setup_app", "enable_app", "sign_zome_call"];
```

Comment:
The `setup_app` command replaces the need for `install_app`, `connect`, `is_app_installed`, and `ensure_app_websocket`.